documentation/src/docs/asciidoc/link-attributes.adoc

```diff
@@ -65,6 +65,7 @@ endif::[]
 :RepetitionInfo:                    {javadoc-root}/org/junit/jupiter/api/RepetitionInfo.html[RepetitionInfo]
 :TestExecutionExceptionHandler:     {javadoc-root}/org/junit/jupiter/api/extension/TestExecutionExceptionHandler.html[TestExecutionExceptionHandler]
 :TestInfo:                          {javadoc-root}/org/junit/jupiter/api/TestInfo.html[TestInfo]
+:TestInstanceFactory:               {javadoc-root}/org/junit/jupiter/api/extension/TestInstanceFactory.html[TestInstanceFactoryh]
```

Comment:
Remove trailing "h" in link name.

TestInstanceFactoryh --> TestInstanceFactory